app/components/cookiesyncer-frame.js

```diff
@@ -0,0 +1,11 @@
+import {inject as service} from '@ember/service';
+import {computed} from '@ember/object';
+import config from '../config/environment';
+import Component from '@ember/component';
+
+export default Component.extend({
+	currentUser: service(),
+	tagName: '',
+	servicesDomain: computed(() => config.APP.servicesExternalHost),
+	isCookieSet: (typeof $.cookie('tracking-opt-in-status') === 'undefined' ? false : true)
```

Comment:
We're migrating away from jQuery. Please use
```window.Cookies.get('tracking-opt-in-status')```

Reply:
Also, this will fail in FastBoot because there is no `$` or `window` on server. This should be a computed property. You can use this in FastBoot:
```
fastboot: service()
...
this.get('fastboot.request.cookies')
```